web/src/app/core/user.service.ts

```diff
@@ -1,29 +1,52 @@
-import { Injectable } from "@angular/core";
+import { Injectable } from '@angular/core';
+import { HttpClient } from '@angular/common/http';
+import { HttpHeaders } from '@angular/common/http';
+import { Observable } from 'rxjs/observable';
+import 'rxjs/add/operator/map';
 import 'rxjs/add/operator/toPromise';
 import { AngularFirestore } from '@angular/fire/firestore';
 import { AngularFireAuth } from '@angular/fire/auth';
 import * as firebase from 'firebase/app';
+import { User } from 'firebase/app';
+import { FirebaseUserModel } from './user.model';
+
+const httpOptions = {
+  headers: new HttpHeaders({
+    'Content-Type':  'application/json',
+  })
+};
 
 @Injectable()
 export class UserService {
 
+  url:string = 'http://localhost:63073/api/user/' 
+
   constructor(
    public db: AngularFirestore,
-   public afAuth: AngularFireAuth
- ){
+   public afAuth: AngularFireAuth,
+   private http: HttpClient
+ ) {
  }
 
+  getUser(id : string):Observable<FirebaseUserModel>{
+    return this.http.get<FirebaseUserModel>(this.url + id);
+  }
+
+  updateUser(user : FirebaseUserModel):Observable<FirebaseUserModel>
+  {
```

Comment:
Just nitpicking here but it would be better to stick to a single style and move this brace to the same line as the method signature.